.eslintrc.json

```diff
@@ -1,4 +1,5 @@
 {
+    "root": true,
```

Comment:
This is just a small thing I need to add to avoid conflicts with my global ``.eslintrc``